### ISSUE TITLE
Loss is decrease but SER is increase

### ISSUE THREAD

**Issue by liangzhenlin123** [HawkAaron/RNN-Transducer#19]
Hello, I used RNNT training on the Chinese speech recognition library of more than 300 hours (the encoder did pretrain, but the decoder is a random initialization parameter). After training dozens of epoch, the loss first quickly dropped from more than 1000 to 60. Then slowly dropped to more than 20, but the SER of inference has risen from 2 to 20. Is this normal? It seems that you mentioned this phenomenon elsewhere.
Thank you very much!
